ProjectLibertyLabs/social-app-template
# 144 refresh feed when backend sees new post

# Purpose
The goal of this PR is to refresh feed when the backend webhook sees a new post or reply.

Closes #144 

## Solution

- Add a call in the webhook controller that tells the frontend to fetch the new post or reply.

@JoeCap08055 @mattheworris 


### Change summary
* removed timeouts that were initially being used to control the loading indicators
* Add SSE to communicate that the post has been published to the frontend
* Call fetch data on frontend to update the UI


## Steps to Verify
1. Create new post
2. See loading until new post appears in feed
3. Create new reply
4. See loading until new reply appears in feed


## Additional details / screenshot

https://github.com/user-attachments/assets/9d8e5f72-d5ca-4c04-a87c-547406e5d540
